### ISSUE TITLE
Foundations & Javascript (NodeJS): Update Scrim links to newer versions

### ISSUE THREAD

**Issue by mathdebate09** [TheOdinProject/curriculum#28259]
### Checks

- [x] This is not a duplicate of an existing issue (please have a look through our [open issues list](https://github.com/TheOdinProject/curriculum/issues) to make sure)
- [X] I have thoroughly read and understand [The Odin Project Contributing Guide](https://github.com/TheOdinProject/.github/blob/main/CONTRIBUTING.md)
- [ ] Would you like to work on this issue?

### Describe your suggestion

Scrimba recently had a major update in how their website is managed rendering a lot of links invalid, they do redirect to the newer pages but the links don't seem to work, I've compiled a list of fixes necessary with newer links in [the spreadsheet](https://docs.google.com/spreadsheets/d/13I4gZP_xfdbGX3hGtQFenyyXvwYKx-PwV1lT_zicN0U).

If maintainers approve we can start assigning these to contributors, I'll keep the issue updated if they've been assigned/merged by maintainers yet, or not.

### Guidance

- Comment on this issue to let us know you'd like to get a section assigned to you. You can also ask for a specific one.
- Please wait until a maintainer assigns you before starting to work on this issue to avoid multiple people working on the same lessons.
- When you have been assigned a section, please refer to the suggested [link text changes spreadsheet](https://docs.google.com/spreadsheets/d/13I4gZP_xfdbGX3hGtQFenyyXvwYKx-PwV1lT_zicN0U) for instructions on what changes need to be made for your assigned lesson(s).
  - For most of these, you should just be able to copy and paste the updated link to replace the existing link shown in the spreadsheet.
  - You are not required to fix any unrelated, preexisting linting errors such as list styles/indentation. Existing linting errors may cause certain tests to fail when you open a PR. You can ignore this and a maintainer will let you know if further action is required.
- Make sure to read the [general contributing guide](https://github.com/TheOdinProject/.github/blob/main/CONTRIBUTING.md) as well as the [curriculum repo contributing guide](https://github.com/TheOdinProject/curriculum/blob/main/CONTRIBUTING.md) before starting your first contribution.
- Please add the following to your PR in the Issues section: `Related to #XXXXX` , since this issue is divided into sections, individual PRs should NOT INCLUDE the `closes` keyword.

#### Foundations

- [ ] [Intro to CSS](https://www.theodinproject.com/lessons/foundations-intro-to-css#additional-resources) && [The Cascade](https://www.theodinproject.com/lessons/foundations-the-cascade#additional-resources)
- [ ] [The Box Model](https://www.theodinproject.com/lessons/foundations-the-box-model#additional-resources) && [Block and Inline](https://www.theodinproject.com/lessons/foundations-block-and-inline#additional-resources)
- [ ] [Introduction to Flexbox](https://www.theodinproject.com/lessons/foundations-introduction-to-flexbox#additional-resources) && [Growing and Shrinking](https://www.theodinproject.com/lessons/foundations-growing-and-shrinking#additional-resources)
- [ ] [Axes](https://www.theodinproject.com/lessons/foundations-axes#additional-resources) && [Alignment](https://www.theodinproject.com/lessons/foundations-alignment#additional-resources)

#### Javascript (NodeJS)

- [ ] [Objects and Objects Constructors](https://www.theodinproject.com/lessons/node-path-javascript-objects-and-object-constructors#additional-resources) && [Asynchrounous Code](https://www.theodinproject.com/lessons/node-path-javascript-asynchronous-code#additional-resources) && [Async and Await](https://www.theodinproject.com/lessons/node-path-javascript-async-and-await#additional-resources)

### Path

Foundations, Node / JS

### Lesson Url

https://www.theodinproject.com/paths/foundations/courses/foundations
https://www.theodinproject.com/paths/full-stack-javascript/courses/javascript

[Edit: Verbiage/Spell checks]

**Comment by SumonGFC** [comment]
You can assign me one

**Comment by theSamyak** [comment]
Hi,
I'd like to volunteer for this task. Please assign me a section.

**Comment by TheSeagullDev** [comment]
I would love to work on a section in foundations.

**Comment by MaoShizhong** [comment]
Thanks @mathdebate09.
I have checked all of the highlighted lessons and their Scrimba links and found all of them automatically redirect to the correct and working Scrimba course, meaning the current links in the lesson (for now) do not actually need to be changed. This is in contrast to one or two previous links that actually did redirect to the wrong course entirely, and therefore those ones had to be changed.

i.e. I am unable to replicate any actual issue with any of these links on my devices, as they all end up taking you to the correct Scrimba courses. Can others replicate this?

**Comment by mathdebate09** [comment]
Yes most of them redirect to v1/v2 links, the thought just occurred to me incase if they in future became inaccessible, rather just update the links for now itself, the links work perfect for me as well.

**Comment by SumonGFC** [comment]
For reference, the only site that did not redirect me to the v2 site was [The Cascade](https://www.theodinproject.com/lessons/foundations-the-cascade#additional-resources)
which redirected me to the v1 site

**Comment by MaoShizhong** [comment]
v1 or v2 won't matter, that's a Scrimba thing. Whether it's v1 or v2 Scrimba, as long as we land on the correct video and that video works, we're happy. A couple of Scrimba links before actually didn't take us to the correct video (IIRC, some took us to some sign up page or some other generic dashboard), so those had to be changed. Since these ones take us to the correct videos, we can leave them. Should this change in the future, someone can open a new issue/PR to replace that broken link

**Comment by mathdebate09** [comment]
> v1 or v2 won't matter, that's a Scrimba thing. Whether it's v1 or v2 Scrimba, as long as we land on the correct video and that video works, we're happy. A couple of Scrimba links before actually didn't take us to the correct video (IIRC, some took us to some sign up page or some other generic dashboard), so those had to be changed. Since these ones take us to the correct videos, we can leave them. Should this change in the future, someone can open a new issue/PR to replace that broken link

Yes that works the best
